### PR TITLE
Translations files for dashboard and components

### PR DESCRIPTION
**Pull request by JordanSh**
https://github.com/elastic/security-team/issues/2582 Added translations files accordingly

### CODE REVIEW

File: x-pack/plugins/cloud_security_posture/public/components/unknown_route.tsx [hunk 1]
```diff
@@ -18,7 +19,7 @@ export const UnknownRoute = React.memo(() => (
       iconType="logoElastic"
       title={
         <p>
-          <FormattedMessage id="xpack.csp.unknownRoute" defaultMessage="Page not found" />
+          <FormattedMessage id="xpack.csp.unknownRoute" defaultMessage={TEXT.PAGE_NOT_FOUND} />
```
Comment (orouz) [inline x-pack/plugins/cloud_security_posture/public/components/unknown_route.tsx:22]: `TEXT.PAGE_NOT_FOUND` already defined a key: `xpack.csp.page_not_found`

it seems this registers another key with the above key's value.

i think we can use `TEXT.PAGE_NOT_FOUND` directly?

File: x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/translations.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  defaultMessage: 'Cloud Posture',
+});
+
+export const TOP_5_CHART_TITLE = i18n.translate('xpack.csp.top_5_chart_title', {
+  defaultMessage: 'Top 5 Resources Types At Risk',
+});
+
+export const TOP_5_CHART_DESCRIPTION = i18n.translate('xpack.csp.top_5_chart_description', {
+  defaultMessage: 'Non compliant first',
+});
```
Comment (orouz) [inline x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/translations.ts:20]: maybe rename the variable to match the text better?

File: x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/translations.ts [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  }
+);
+
+export const SCORE_PER_CLUSTER_CHART_DESCRIPTION = i18n.translate(
+  'xpack.csp.score_per_cluster_chart_title',
+  {
+    defaultMessage: 'Non compliant first',
+  }
+);
+
```
Comment (orouz) [inline x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/translations.ts:35]: seems like we already have this text as `TOP_5...` 

is this intentional?

**Review by orouz — DISMISSED** [review]
see comments

File: x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/translations.ts (hunk 3 above)
Comment (JordanSh) [reply x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/translations.ts:35]: was, but your suggestion is better. changed

**Review by orouz — APPROVED** [review]

**Pull request merged**